src/AWSLambda/Handler.hs

```diff
@@ -9,15 +9,17 @@ Entry point for AWS Lambda handlers deployed with @serverless-haskell@ plugin.
 -}
 module AWSLambda.Handler
   ( lambdaMain
+  , runMain
```

Comment:
What's the reason for exporting this? It should be named better to avoid confusion.

Reply:
I thought it might be useful to some users to have a lower-level interface exposed in case they want to use something other than JSON values. I agree the name should be better, but I don't have a good idea. Maybe `rawMain` or `lambdaMainBS`?

Reply:
`lambdaMainRaw` or `lambdaMainBS` are good.

Reply:
I'm not sure AWS Lambda will send anything other than JSON values, but perhaps you are using a different JSON parser.

Reply:
Done